### PR TITLE
Use manifest for app config in build process

### PR DESCRIPTION
**Pull request by jbalboni**
There are a lot of things to update when creating an application, and they're mostly all based on a couple pieces of information. It seems like we could push some of this into config files and have the build process find and read them.

### CODE REVIEW

File: src/js/claims-status/manifest.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "appName": "Claims Status",
+  "entryPoint": "./src/js/claims-status/claims-status-entry.jsx",
+  "bundleName": "claims-status",
+  "rootUrl": "/track-claims",
+  "production": true
+}
```
Comment (ncksllvn) [inline src/js/claims-status/manifest.json]: So with the `entryPoint` property, we could ditch the giant hardcoded `entryFiles` array in the webpack config?

Comment (jbalboni) [reply src/js/claims-status/manifest.json]: I think so, yes.

File: script/build.js [hunk 2]
```diff
@@ -431,34 +446,19 @@ if (options.watch) {
...
-        { from: '^/letters(.*)', to: '/letters/' },
-        { from: '^/pension/application/527EZ(.*)', to: '/pension/application/527EZ/' },
-        { from: '^/burials-and-memorials/application/530(.*)', to: '/burials-and-memorials/application/530/' },
-        { from: '^/burials-and-memorials/pre-need/form-10007-apply-for-eligibility(.*)', to: '/burials-and-memorials/pre-need/form-10007-apply-for-eligibility/' },
-        { from: '^/employment/vocational-rehab-and-employment/application/chapter31(.*)', to: '/employment/vocational-rehab-and-employment/application/chapter31/' },
-        { from: '^/employment/vocational-rehab-and-employment/application/chapter36(.*)', to: '/employment/vocational-rehab-and-employment/application/chapter36/' },
-        { from: '^/veteran-id-card/apply(.*)', to: '/veteran-id-card/apply/' },
-        { from: '^/disability-benefits/526/apply-for-increase(.*)', to: '/disability-benefits/526/apply-for-increase/' },
+        ...appRewrites,
         { from: '^/(.*)', to(context) { return context.parsedUrl.pathname; } }
```
Comment (ncksllvn) [inline script/build.js]: Awesome 💥

File: src/js/rx/manifest.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
+  "appName": "Prescription Refill",
+  "entryFile": "./rx-entry.jsx",
+  "entryName": "rx",
+  "rootUrl": "/health-care/prescriptions",
+  "production": true
+}
```
Comment (ncksllvn) [inline src/js/rx/manifest.json]: So when one of these files is edited/added, Jenkins will process it and automatically submit a PR to the rev proxy? Lol

**Review by rianfowler — APPROVED** [review]
very nice

**Comment by bshyong** [comment]
💯

**Review by bshyong — APPROVED** [review]

**Review by ncksllvn — APPROVED** [review]
Awesome 🐳

**Pull request merged**